logging/zerolog/README.md

```diff
@@ -0,0 +1,89 @@
+# Kitex zerolog (This is a community driven project)
+
+## Introduction
+
+This is a logger library that uses [zerolog](https://github.com/rs/zerolog) to implement the [Kitex logger interface](https://www.cloudwego.io/docs/kitex/tutorials/basic-feature/logging/), work together with kitex [obs-opentelemetry](https://github.com/kitex-contrib/obs-opentelemetry)
+
+## Usage
+
+Download and install it:
+
+```go
+go get github.com/kitex-contrib/obs-opentelemetry/logging/zerolog
+```
+
+Import it in your code:
+
+```go
+import kitexzerolog github.com/kitex-contrib/obs-opentelemetry/logging/zerolog
+```
+
+### Set logger impl
+
+```go
+package main
+
+import (
+    "github.com/rs/zerolog/log"
+    "github.com/cloudwego/kitex/pkg/klog"
+    kitexzerolog "github.com/kitex-contrib/obs-opentelemetry/logging/zerolog"
+)
+
+func main() {
+    logger := kitexzerolog.NewLogger()
+    klog.SetLogger(logger)
+    klog.SetLevel(klog.LevelDebug)
+
+    // OR / AND using global logger
+    log.Logger = *logger.Logger()
+}
+```
+
+> We provide some methods to help you customize logger
+
+| Configuration              | Description                                                                   |
+| -------------------------- | ----------------------------------------------------------------------------- |
+| WithLogger                 | [Logger](https://pkg.go.dev/github.com/rs/zerolog#Logger)                       |
+| WithTraceErrorSpanLevel    | trace error span level option                                                 |
+| WithRecordStackTraceInSpan | record stack track option                                                     |
+
+### Log with context
+
+```go
+// Echo implements the Echo interface.
+func (s *EchoImpl) Echo(ctx context.Context, req *api.Request) (resp *api.Response, err error) {
+    klog.CtxDebugf(ctx, "echo called: %s", req.GetMessage())
+    return &api.Response{Message: req.Message}, nil
+}
+```
+
+### Log with global logger
```

Comment:
This section is not part of our library's content and should not be included in our documentation.

Reply:
The reason why I added it here as an example is because I want to let users know that the hook can also affect when they use the global logger (not only via our `klog`).

Reply:
Okay, understood.